app/helpers/passport.js

```diff
@@ -1,17 +1,20 @@
 'use strict';
+
+var request = require('request');
 var GitHubStrategy = require('passport-github').Strategy;
+var BearerStrategy = require('passport-http-bearer').Strategy;
 
 var User = require('../models/user');
+var Account = require('../models/account');
 var config = require('../../config');
 
-// expose this function to our app using module.exports
 module.exports = function(passport) {
     passport.serializeUser(function(user, done) {
```

Comment:
It's used for automatically serializing and deserializing the user into the session. [Required by passport](https://www.npmjs.com/package/passport), otherwise it will throw a serialization error